content/docs/connect/connect-from-any-app.md

```diff
@@ -77,6 +75,11 @@ The **Connection Details** widget on the **Neon Dashboard** also provides connec
 
 Our *Guides* documentation also provides connection examples.
 
+## Connection notes
+
+- Some older client libraries and drivers, including older `psql` executables, are built without [Server Name Indication (SNI)](/docs/reference/glossary#sni) support and require a workaround. For more information, see [Connect from older clients](/docs/connect/connectivity-issues).
+- Some Java-based tools that use the pgJDBC driver for connecting to PostgreSQL, such as DBeaver, DataGrip, and CLion, do not support including a role name and password in a database connection string. When you find that a connection string is not accepted, try entering the role name and password values in the appropriate fields in the tool's UI when configuring a connection to Neon.
```

Comment:
I recommend to provide an example of connection string for such drivers

Reply:
@vadim2404 Since this is about connecting from Java-based GUIs that use pgJDBC, I added a DBeaver example to the Connect a  GUI or IDE topic:
https://neon-next-git-dprice-add-secure-connection-topic-neondatabase.vercel.app/docs/connect/connect-postgres-gui#connect-to-the-database
I linked to this example from the Connection note.